SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSystem.cpp

```diff
@@ -1694,6 +1696,44 @@ USpatialActorChannel* ActorSystem::TryRestoreActorChannelForStablyNamedActor(AAc
 	return Channel;
 }
 
+void ActorSystem::InvokeRepNotifies()
+{
+	for (FObjectRepNotifies ObjectRepNotifies : RepNotifiesToSend)
+	{
+		UObject* Object = ObjectRepNotifies.Object.Get();
```

Comment:
could be const?

Reply:
can't be done due to being arg to Channel->PostReceiveSpatialUpdate unfortunately